policyengine_us/variables/gov/states/nm/tax/income/credits/cdcc/nm_cdcc.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class nm_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico dependent child day care credit"
+    defined_for = StateCode.NM
```

Comment:
And remove the eligibility test inside
```suggestion
    defined_for = "nm_cdcc_eligible"
```